index.html

```diff
@@ -11,22 +11,32 @@
 <body>
     <!--Header Starts Here-->
     <header class="header-container">
-        <div class="header-top">
+        <nav class="nav-top">
             <a href="#" class="header-logo">Welcome</a>
-            <button class="hamburger" type="button">
-                <i class="fa fa-solid fa-bars"></i>
+            <button class="hamburger" id="menu-bar" type="button">
+               <i class="fa fa-solid fa-bars"></i>
             </button>
-            <nav>
-                <ul class="header-nav-desktop">
+            <ul class="header-nav-desktop">
                     <li><a href="">Portifolio</a></li>
                     <li><a href="">About</a></li>
                     <li><a href="">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="mobile-menu" id="menu-mobile">
+            <div class="menu-header">
+                <span class="close">&times;</span>
+            </div>
+            <div class="menu-body">
+                <ul>
+                    <li><a href="#work-id">Portfolio</a></li>
+                    <li><a href="#about-id">About</a></li>
+                    <li><a href="#contact-id">Contact</a></li>
```

Comment:
- [ ] it would be better if the header shouldn't appear when the menu is open in order to match the Figma design 
- Yours 
<img width="377" alt="Screenshot 2022-11-14 at 15 19 29" src="https://user-images.githubusercontent.com/61790587/201683607-f47f9296-b45d-40e3-9538-d893c1a77b74.png">
- Figma 
<img width="274" alt="Screenshot 2022-11-14 at 15 21 09" src="https://user-images.githubusercontent.com/61790587/201683692-2758c04f-0602-4619-bd8c-dc1ba058deef.png">